application-templates/starter/src/components/channel-details/channel-details-form.js

```diff
@@ -0,0 +1,104 @@
+import PropTypes from 'prop-types';
+import { useFormik } from 'formik';
+import { useIntl, FormattedMessage } from 'react-intl';
+import { useApplicationContext } from '@commercetools-frontend/application-shell-connectors';
+import LocalizedTextField from '@commercetools-uikit/localized-text-field';
+import TextField from '@commercetools-uikit/text-field';
+import Spacings from '@commercetools-uikit/spacings';
+import { useIsAuthorized } from '@commercetools-frontend/permissions';
+import { ErrorMessage } from '@commercetools-uikit/messages';
+import SelectField from '@commercetools-uikit/select-field';
+import { PERMISSIONS } from '../../constants';
+import { CHANNEL_ROLES } from './constants';
+import validate from './validate';
+import messages from './messages';
+
+const getRoleOptions = (intl) =>
+  Object.keys(CHANNEL_ROLES).map((key) => ({
+    label: intl.formatMessage(messages[key]),
```

Comment:
I wouldn't translate these values, just keep them as-is.

```suggestion
    label: CHANNEL_ROLES[key],
```